matplotlib-stubs/pyplot.pyi

```diff
@@ -104,13 +113,88 @@ def figure(
     clear: bool = ...,
 ) -> Figure: ...
 def subplots_adjust(
-    left: Optional[float] = ...,
-    bottom: Optional[float] = ...,
-    right: Optional[float] = ...,
-    top: Optional[float] = ...,
-    wspace: Optional[float] = ...,
-    hspace: Optional[float] = ...,
+    left: Optional[_Numeric] = ...,
+    bottom: Optional[_Numeric] = ...,
+    right: Optional[_Numeric] = ...,
+    top: Optional[_Numeric] = ...,
+    wspace: Optional[_Numeric] = ...,
+    hspace: Optional[_Numeric] = ...,
 ) -> None: ...
 def close(fig: Union[Figure, Literal["all"]]) -> None: ...
 def clf() -> None: ...
+def plot(
+    x: _Data,
+    y: _Data,
+    *,
+    color: Optional[str] = ...,
+    label: str = ...,
+    linestyle: Literal["-", "--", "-.", ":", ""] = ...,
+    marker: str = ...,
+    markerfacecolor: str = ...,
+    markersize: _Numeric = ...,
+    scalex: bool = ...,
+    scaley: bool = ...,
+    zorder: _Numeric = ...,
+) -> None: ...
 def show() -> None: ...
+def xlim(
+    left: _Numeric = ...,
+    right: _Numeric = ...,
+    emit: bool = ...,
+    auto: Optional[bool] = ...,
+    xmin: _Numeric = ...,
+    xmax: _Numeric = ...,
+) -> Tuple[_Numeric, _Numeric]: ...
+def ylim(
+    bottom: _Numeric = ...,
+    top: _Numeric = ...,
+    emit: bool = ...,
+    auto: Optional[bool] = ...,
+    ymin: _Numeric = ...,
+    ymax: _Numeric = ...,
+) -> Tuple[_Numeric, _Numeric]: ...
+def xticks(
+    ticks: Optional[_NumericArray] = ...,
+    labels: Optional[_StrArray] = ...,
+    *,
+    fontdict: Optional[Dict[str, Union[str, int]]] = ...,
+    minor: bool = ...,
+    **kwargs: Any,
```

Comment:
In general I don't like `**kwargs` in type stubs. they make the whole affair kind of pointless...